js/generating-similar-elements.js

```diff
@@ -0,0 +1,54 @@
+import {createPosts} from './data.js';
+
+const mapCanvas = document.querySelector('#map-canvas');
+const similarPostTemplate = document.querySelector('#card').content.querySelector('.popup');
+const similarPosts = createPosts();
+
+const getPhotos = (randomPhoto, postElement) => {
+  const photosContainer = postElement.querySelector('.popup__photos');
+  const photoList = postElement.querySelector('.popup__photo');
+  photoList.remove();
+  randomPhoto.forEach((photo) => {
+    const newPhotoElement = photoList.cloneNode(true);
+    for (let i = 0; i < randomPhoto.length; i++){
+      newPhotoElement.setAttribute('src', photo);
+      photosContainer.appendChild(newPhotoElement);
+    }
+  });
```

Comment:
```suggestion
    const newPhotoElement = photoList.cloneNode(true);
    
    for (let i = 0; i < randomPhoto.length; i++){
      newPhotoElement.setAttribute('src', photo);
      photosContainer.appendChild(newPhotoElement);
    }
  });
```

ну и фор по хорошему бы на форыч поменять.